docs/contributing/tasks.md

```diff
@@ -63,9 +63,9 @@ npm scripts are defined in `package.json`. These trigger a number of Gulp tasks.
 
 ## Gulp tasks
 
-Gulp tasks are defined in `gulpfile.js` and .`/tasks/gulp/` folder.
+Gulp tasks are defined in `gulpfile.mjs` and .`/tasks/` folder.
```

Comment:
Am I missing something about why the full stop is outside of the `<code>` element here?

Also, can we tweak it to either "and the .`/tasks/` folder" or "and .`/tasks/`"?

Reply:
Hmm how strange, I didn't even notice. It's been in since 2017 https://github.com/alphagov/govuk-frontend/commit/2e64f5b7ca40df032b7d4611a2121553dadbca5e#diff-06cac645496a5082a65db61580c0d7b2d3890e8d164ca7f4d1193e8a28f76a68R92

To avoid lots of rebasing, mind if we add a follow up typo PR?

Reply:
@domoscargin Full stop removal included in https://github.com/alphagov/govuk-frontend/pull/3384/commits/edeb6d705ff0c6627a8d500e1f63c88d1c96e89b